### DISCUSSION TITLE
Option for a better recognizable blue HA navigation bar selector (for visually impaired)

### DISCUSSION THREAD

**Discussion by typxxi** [home-assistant/frontend#20131] [Accessibility]
Hello,

we have now a visually impaired family member that needs to use HA.

We have currently an issue with the horizontal bar cause he can not recognise which of the tabs is the currently active one as seen here the °C tab.

![image](https://github.com/home-assistant/frontend/assets/20662430/657c5bc0-d924-4db4-ba6d-8337a8551ffe)

There is only a small white indicator line at the bottom of the bar. Its height is way too low for him to be recogniced. Also the text colours are not different enough, pretty much the same white, just the current tab is a bit whiter than the others which are more white greyish.

The easiest way would be an adjustable height for the white indicator on the bottom of the blue HA bar which would not bother other users who do not need that.

We have tried dark mode but that did not work out for other reasons so there should be a solution to enhance / highlight the selected  tab way better than currently in the white / creme style.

**Comment by ildar170975** [comment]
A possible solution using card-mod was provided in Community in your thread - https://community.home-assistant.io/t/better-setup-for-visually-impaired-of-the-blue-ha-navigation-bar/705706/4?u=ildar_gabdullin.

![1q](https://github.com/home-assistant/frontend/assets/71872483/cdf6aaa8-4ac1-4935-adf5-e90c5e95426d)
